test/namespace_assertion.go

```diff
@@ -3,17 +3,22 @@ package test
 import (
 	"context"
 
+	toolchainv1alpha1 "github.com/codeready-toolchain/api/pkg/apis/toolchain/v1alpha1"
+
 	"github.com/codeready-toolchain/toolchain-common/pkg/test"
+
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
-
 	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/api/meta"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/types"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+type ResourceCriterion func(obj runtime.Object) bool
+
```

Comment:
This is not used if I'm not mistaken
```suggestion
```

Reply:
good catch, fixed as part of: ad15c624078af2b4d102df7ed1304e6483044976